esx_service/cli/vmdkops_admin.py

```diff
@@ -121,6 +121,9 @@ def commands():
                                 'attached-to', 'policy', 'capacity', 'used',
                                 'fstype', 'access', 'attach-as'],
                     'metavar': 'Col1,Col2,...'
+                },
+                '--tenant' : {
+                    'help': 'Displays only VMs for a given tenant'
```

Comment:
Should --tenant be a part of the ls command? It seems to be listing volumes by default.

Instead we could have like:

vmdkops_admin with commands like
a. volume - ls, create, rm, set
b. tenant - ls, create, rm, set
c. <anymore>

which matches with esxcli type command options where we have broad categories within which the same CRUD options are available (list, create, delete, ....)